feather/server/src/config.rs

```diff
@@ -11,19 +11,21 @@ use crate::{favicon::Favicon, Options};
 const DEFAULT_CONFIG: &str = include_str!("../config.toml");
 
 /// Loads the config, creating a default config if needed.
-pub fn load(path: &str) -> anyhow::Result<Config> {
+pub fn load(path: &str) -> anyhow::Result<(Config, bool)> {
```

Comment:
I think the result should be wrapped in a `struct`, since its not clear what the `bool` indicates.